scripts/prepare.sh

```diff
@@ -3,16 +3,11 @@ set -euo pipefail
 
 eval RAINBOW_DIR=~/.rainbow
 
-export LOCAL_CORE_SRC
-export LOCAL_NEARUP_SRC
-
 eval CORE_SRC=~/.rainbow/core
```

Comment:
This lines delete environment variables which are being passed to the exe
So instead of using already existing CORE repo which has already built neard version it clones
and makes a clean build which is slow